packages/ipx/src/input/hybrid.ts

```diff
@@ -0,0 +1,36 @@
+import { Stats } from 'fs-extra'
+import BaseInputAdapter from './BaseInputAdapter'
+import FSAdapter from './fs'
+import RemoteAdapter from './remote'
+
+function isRemote(url: string) {
+  return url.startsWith('http');
+}
+
+export default class HybridAdapter extends BaseInputAdapter {
+  fsAdapter!: FSAdapter
+  remoteAdapter!: RemoteAdapter
+
+  init() {
+    this.fsAdapter = new FSAdapter(this.ipx)
+    this.remoteAdapter = new RemoteAdapter(this.ipx)
+  }
+  
+  async stats (src: string): Promise<Stats | false> {
+    if (isRemote(src)) {
```

Comment:
We should support prefix for input URLs (in core). It is not job of an input to dispatch.

Reply:
If so, we must create a validate api in inputs and support multiple inputs in config